src/templates/fr/2019/table_of_contents.html

```diff
@@ -0,0 +1,254 @@
+  border-bottom: none;
+}
+
+.part-name,
+.chapter {
+  padding: 20px 0;
+}
+
+.chapters {
+  padding-left: 20px;
+}
+
+.chapter .todo {
+  opacity: 0.3;
+}
+.chapter a {
+  text-decoration: none;
+  color: #1A2B49;
+}
+.chapter a:hover,
+.chapter a:focus {
+  text-decoration: underline;
+}
+@media (max-width: 600px) {
+  .contents-wrapper {
+    display: block;
+  }
+  .parts {
+    max-width: 100%;
+  }
+  .characters {
+    display: none;
+  }
+}
+</style>
+{% endblock %}
+
+{% block main %}
+<section class="main">
+  <h1 class="title title-lg">Table des matières</h1>
+
+  <section>
+    <h2>Avant-propos</h2>
+
+    <p>
+      Le Web ouvert est un réseau de technologies étonnamment complexe et évolutif. Des industries et des carrières entières sont bâties sur le Web et dépendent de son écosystème dynamique pour réussir. Aussi crucial que soit le Web, il a été étonnamment difficile de comprendre comment il fonctionne. Depuis 2010, la mission du projet HTTP Archive est de suivre comment le web est construit, et il a fait un travail incroyable. Cependant, une lacune s’est révélée particulièrement difficile à combler : donner un sens aux données que le projet HTTP Archive a collectées et permettre à la communauté de comprendre facilement les performances du Web. C’est là qu’intervient le Web Almanac.
+    </p>
+
+    <p>
+      La mission de l’Almanach Web est de regrouper les trésors d’informations qui, autrement, ne seraient accessibles qu’aux intrépides explorateurs de données, de manière à ce qu’elles soient faciles à comprendre. Nous y sommes parvenu·e·s avec l’aide d’expert·e·s de l’industrie qui comprennent les données et nous informent sur ce qu’elles signifient. Chacun des 20 chapitres du Web Almanac se concentre sur un aspect spécifique du Web, et chacun d’entre eux a été rédigé et révisé par des expert·e·s dans leur domaine. La force du Web Almanac découle directement de l’expertise des personnes qui l'écrivent.
+    </p>
+
+    <p>
+      Un grand nombre de résultats du Web Almanac méritent d'être salués, mais c’est aussi un rappel important du travail qu’il reste à faire pour offrir une expérience utilisateur de grande qualité. Les analyses fondées sur les données présentées dans chaque chapitre sont une forme de responsabilisation que nous partageons tous et toutes pour développer un meilleur Web. Il ne s’agit pas de faire honte à celles et ceux qui se trompent, mais de mettre en lumière le chemin menant aux bonnes pratiques pour qu’il y ait une façon claire et juste de faire les choses. Avec l’aide continue de la communauté Web, nous espérons en faire une tradition annuelle, de sorte que chaque année, nous puissions suivre nos progrès et faire des ajustements de cap si nécessaire.
+    </p>
+
+    <p>
+      Il y a tant à apprendre dans ce rapport, que nous vous invitons à l’explorer dès à présent et à partager vos découvertes avec la communauté afin que nous puissions collectivement améliorer notre compréhension de l'état du Web.
```

Comment:
Des fois des apostrophes, des fois des ', autant avoir la vraie apostrophe partout ;)

Reply:
My bad, I used `(\w)'(\w)` instead of `([A-zÀ-ú])'([A-zÀ-ú])` to detect infractions. I missed diacritics… that's ok now.